cmd/soroban-cli/src/events.rs

```diff
@@ -0,0 +1,182 @@
+use std::io;
+use std::io::Write;
+
+use clap::Parser;
+use hex::FromHexError;
+use soroban_env_host::xdr::{ReadXdr, ScVal};
+use termcolor::{Color, ColorChoice, ColorSpec, StandardStream, WriteColor};
+
+use crate::rpc::Event;
+use crate::utils;
+use crate::{rpc, rpc::Client};
+use crate::{HEADING_RPC, HEADING_SANDBOX};
+
+#[derive(Parser, Debug)]
+#[clap()]
+pub struct Cmd {
+    /// The ledger range to pull events from
+    start_ledger: u32,
```

Comment:
Is this a block height? Also should it be possible to have open ranges, e.g. `start_ledger: 10` --> til the end?

Lastly is there a max range?

Reply:
The [ledger sequence number](https://developers.stellar.org/docs/glossary/ledger/) is akin to the block height, yeah.

According to the [referenced API spec](https://docs.google.com/document/d/1TZUDgo_3zPz7TiPMMHVW_mtogjLyPL0plvzGMsxSz6A/edit#heading=h.t03hji406t76), open ranges aren't possible (yet, for v0), and the max range is a `TODO` item at the moment. It's likely that there will be one!

Reply:
Ah well then perhaps it would be helpful to add because currently it's unclear what the type of this is from the help text.

Reply:
I think even including the link would be great. Perhaps this one:

https://developers.stellar.org/docs/encyclopedia/ledger-headers#ledger-sequence

Reply:
Ah, good call. This is common nomenclature in the Stellar world, but cross-references are always helpful for people exploring this for the first time!